### ISSUE TITLE
New add-on to provide .NET Core imagestreams and templates

### ISSUE THREAD

**Issue by gbraad** [minishift/minishift-addons#78]
Related to: https://github.com/minishift/minishift/issues/1819

```
$ oc create --namespace openshift -f https://raw.githubusercontent.com/redhat-developer/s2i-dotnetcore/master/dotnet_imagestreams.json
$ oc create --namespace openshift -f https://raw.githubusercontent.com/redhat-developer/s2i-dotnetcore/master/templates/dotnet-example.json 
$ oc create --namespace openshift -f https://raw.githubusercontent.com/redhat-developer/s2i-dotnetcore/master/templates/dotnet-runtime-example.json 
$ oc create --namespace openshift -f https://raw.githubusercontent.com/redhat-developer/s2i-dotnetcore/master/templates/dotnet-pgsql-persistent.json
```

**Comment by gbraad** [comment]
Repo:
  * https://github.com/redhat-developer/s2i-dotnetcore

Example:
  * https://github.com/redhat-developer/s2i-dotnetcore-ex

Description:
  * https://access.redhat.com/documentation/en-us/net_core/2.0/html/getting_started_guide/gs_dotnet_on_openshift

**Comment by LalatenduMohanty** [comment]
resolved via #80
